### PR TITLE
[DI] Fix the xml schema

### PR DESCRIPTION
**Pull request by GuilhemN**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | yes
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no 
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd [hunk 1]
```diff
@@ -140,8 +140,8 @@
   </xsd:complexType>
 
   <xsd:complexType name="property" mixed="true">
-    <xsd:choice minOccurs="0" maxOccurs="1">
```
Comment (GuilhemN) [inline src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd]: This one is the default value.

File: src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd [hunk 2]
```diff
@@ -153,8 +153,8 @@
   </xsd:complexType>
 
   <xsd:complexType name="argument" mixed="true">
-    <xsd:choice maxOccurs="unbounded">
```
Comment (GuilhemN) [inline src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd]: This allowed having several anonymous services in the same argument while only the first was considered by the loader.

File: src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd [hunk 3]
```diff
@@ -165,10 +165,9 @@
     <xsd:attribute name="strict" type="boolean" />
   </xsd:complexType>
 
-  <xsd:complexType name="call" mixed="true">
-    <xsd:choice maxOccurs="unbounded">
-      <xsd:element name="argument" type="argument" minOccurs="0" maxOccurs="unbounded" />
-      <xsd:element name="service" type="service" />
```
Comment (GuilhemN) [inline src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd]: This wasn't used.

**Review by nicolas-grekas — APPROVED** [review]
:+1:

**Comment by fabpot** [comment]
Thank you @GuilhemN.

**Pull request merged**